lib/patchelf/version.rb

```diff
@@ -2,5 +2,5 @@
 
 module PatchELF
   # Current gem version.
-  VERSION = '1.0.0'.freeze
+  VERSION = '1.0.1'.freeze
```

Comment:
I would prefer to bump the version in a separated PR (And I also need to draft/push a release for it)